docs/customizations/footer.md

```diff
@@ -0,0 +1,218 @@
+| emailTranscriptButtonStyleProps | [IStyle](https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/src/IStyle.ts) | No | Styles of the email transcript button | [defaultFooterStyleProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/common/defaultStyles/defaultFooterStyleProps.ts) |
+| emailTranscriptButtonHoverStyleProps | [IStyle](https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/src/IStyle.ts) | No | Styles of the email transcript button while hovered | [defaultFooterStyleProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/common/defaultStyles/defaultFooterStyleProps.ts) |
+| audioNotificationButtonStyleProps | [IStyle](https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/src/IStyle.ts) | No | Styles of the audio notification button | [defaultFooterStyleProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/common/defaultStyles/defaultFooterStyleProps.ts) |
+| audioNotificationButtonHoverStyleProps | [IStyle](https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/src/IStyle.ts) | No | Styles of the audio notification button while hovered | [defaultFooterStyleProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/common/defaultStyles/defaultFooterStyleProps.ts) |
+| footerItemFocusStyleProps | [IStyle](https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/src/IStyle.ts) | No | Styles of the footer sub-components while focused | [defaultFooterStyleProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/common/defaultStyles/defaultFooterStyleProps.ts) |
+
+## Sample Scenarios
+
+Below samples are build upon the base sample, which can be found [here](https://github.com/microsoft/omnichannel-chat-widget#example-usage). The code snippets below will only show the changes needed to be added before `ReactDOM.render`.
+
+--------------------------------
+
+### Replacing default sub-components with custom components
+
+<details>
+    <summary>Show code</summary>
+
+```tsx
+...
+const Copyright = () => {
+    return (
+        <div style={{"fontSize":"12px", "fontFamily":"Segoe UI, Arial", "padding":"2px"}}>© Microsoft 2023</div>
+    );
+};
+
+liveChatWidgetProps = {
+    ...liveChatWidgetProps,
+    footerProps: {
+        controlProps: {
+            hideEmailTranscriptButton: true,
+            hideAudioNotificationButton: true,
+            rightGroup: {
+                children: [
+                    <Copyright/>
+                    // Since this is a static elelenmt, alternatively we can use the string format: 
```

Comment:
do we need the comments?, typo in elelenmt

Reply:
yeah it's to explain another way of attaching the component. Fixed typo